Makefile.d/k8s.mk

```diff
@@ -39,6 +39,9 @@ k8s/manifest/update: \
 	    --set initializer.mysql.secret.enabled=true \
 	    --set initializer.redis.secret.enabled=true \
 	    --set initializer.cassandra.secret.enabled=true \
+	    --set agent.minReplicas=5 \
+	    --set gateway.gateway_config.discoverer.agent_client.connection_pool=10 \
+	    --set indexManager.indexer.discoverer.agent_client.connection_pool=10 \
```

Comment:
for index manager we don't need performance about indexing please set it to 2
and please take care about keep alive setting, indexin needs long time connection

Reply:
how about to set `indexManager.indexer.discoverer.agent_client.dial_option.tcp.dialer.keep_alive=5m` ?

Reply:
5minute is not enough how about set it to 15m for maximum but it's upto minUncommitted index length :angel: